pkg/capacity/capacity.go

```diff
@@ -59,7 +60,103 @@ func FetchAndPrint(showContainers, showPods, showUtil, showPodCount, excludeTain
 	printList(&cm, showContainers, showPods, showUtil, showPodCount, showNamespace, output, sortBy, availableFormat)
 }
 
-func getPodsAndNodes(clientset kubernetes.Interface, excludeTainted bool, podLabels, nodeLabels, namespaceLabels, namespace string) (*corev1.PodList, *corev1.NodeList) {
+// Taints can have two possible formats, key1:effect or key1=value1:effect. This function handles both formats and returns each part as a separate string for comparison.
```

Comment:
Nit: I'm used to function comments starting with the name of the function in golang, would recommend using that pattern throughout. Would also slightly prefer wrapping comment lines at 80 chars, but I don't think I've done great at following that here, it's just a common pattern in k8s projects.